volumic_flex93_ultra.xml.fdm_material

```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material">
+  <metadata>
+    <name>
+      <brand>Volumic</brand>
+      <material>TPE</material>

```

Comment:
Typo here. Your Flex93 Ultra material is made out of TPU. TPU happens to be a thermoplastic elastomer as well, but it's commonly known as TPU (even on [your website](https://shop.imprimante-3d-volumic.com/fr/filament/1068-flex93-ultra-750g-bleu.cfm)). Making it TPU again helps people make customised profiles.
```suggestion
      <material>TPU</material>
```

Reply:
ok, fixed